UnitTest.Commands/html/index.html

```diff
@@ -3,24 +3,24 @@
 <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
-    <link rel="stylesheet" href="html/resources/css/bootstrap.min.css" />
-    <link rel="stylesheet" href="html/resources/css/all.css" />
-    <link rel="stylesheet" href="html/resources/css/c3.min.css" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.5.2/css/bootstrap.min.css" />
```

Comment:
External links to to JS or CSS should have SRI attributes, like:
```
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.5.3/css/bootstrap.min.css" integrity="sha512-oc9+XSs1H243/FRN9Rw62Fn8EtxjEYWHXRvjS43YtueEewbS6ObfXcJNyohjHqVKFPoXXUxwc+q1K7Dee6vv9g==" crossorigin="anonymous" />
```

You can pick up those links right from the [cdnjs](https://cdnjs.com/libraries/twitter-bootstrap) page.

Reply:
This isn't right.  My local copy does have the integrity values.  I'm unclear why there is a difference.

Reply:
See how the merged version actually has the where it was available in the already merged and previous branch so again not clear how these are incorrect. 
 Compare https://github.com/microsoft/ApplicationInspector/blob/main/UnitTest.Commands/html/index.html